fms/utils/serialization.py

```diff
@@ -264,6 +274,7 @@ def load_state_dict_into_model(
     source: If the weights in the state dict didn't come from an FMS model,
             `source` specifies which conversion function might be needed.
             See `serialization.list_sources(architecture)`
+    model_config: The config of the model, if it exists, otherwise None
```

Comment:
nit: maybe "if necessary" instead of "if it exists?" most models have some kind of config, but only some models have a state dict loading mechanism that requires a config.